src/dmd/cparse.d

```diff
@@ -606,9 +617,20 @@ final class CParser(AST) : Parser!AST
         switch (token.value)
         {
         case TOK.identifier:
-            if (token.ident is Id.__func__)
+            const id = token.ident.toString();
+            if (id.length > 2 && id[0] == '_' && id[1] == '_')  // leading double underscore
             {
-                addFuncName = true;     // implicitly declare __func__
+                if (token.ident is Id.__func__)
+                {
+                    addFuncName = true;     // implicitly declare __func__
+                }
+                else if (token.ident is Id.builtin_va_arg)
+                {
+                    e = cparseBuiltin_va_arg();
+                    break;
+                }
+                else
```

Comment:
Please use braces for the else branch also. The dmd style is to be consistent across branches (either both have braces or none).

Reply:
That style isn't consistently followed anywhere in dmd.